Chemistry/GetGateCount/3-GetGateCount.csproj

```diff
@@ -16,9 +16,9 @@
     <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="2.1.1" />
     <PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="2.1.1" />
     <PackageReference Include="Microsoft.PowerShell.5.ReferenceAssemblies" Version="1.1.0" NoWarn="NU1701" />
-    <PackageReference Include="Microsoft.Quantum.Chemistry" Version="0.7.1905.3109" />
-    <PackageReference Include="Microsoft.Quantum.Standard" Version="0.7.1905.3109" />
-    <PackageReference Include="Microsoft.Quantum.Development.Kit" Version="0.7.1905.3109" />
+    <PackageReference Include="Microsoft.Quantum.Chemistry" Version="0.8.1906.1704-beta" />
```

Comment:
should we merge this into master?

Reply:
Ah, good point. I'll target a new release/v0.8 branch, then.